cmd/ci-secret-bootstrap/main_test.go

```diff
@@ -3101,168 +3103,509 @@ func TestMutateGlobalPullSecret(t *testing.T) {
 					".dockerconfigjson": []byte(`{"key":"value"}`),
 				},
 			},
-			expectedErr: fmt.Errorf("failed to get token for registry.ci.openshift.org"),
+			expectedErr: fmt.Errorf("failed to get any token"),
 		},
 		{
 			name: "bad original",
 			secret: &coreapi.Secret{
 				Data: map[string][]byte{
 					".dockerconfigjson": []byte(`{
-	"auths": {
```

Comment:
I do not see the change in this section, except the reformatting?
Could we revert them?